react/index.js

```diff
@@ -204,17 +204,20 @@ class ProductSummary extends Component {
     const {
       displayMode,
       product,
+      showBorders,
       name: showFieldsProps
     } = this.props
 
     const containerClasses = classNames(
-      `${productSummary.nameContainer} flex items-start`,
+      'flex items-start',
       {
         'justify-center': displayMode !== 'inline',
         'justify-left w-100': displayMode === 'inline',
         'pv5': displayMode === 'small',
         't-mini pb2': displayMode !== 'normal',
         'pv6': displayMode === 'normal',
+        [`${productSummary.nameContainer}`] : !showBorders,
```

Comment:
You don't need string interpolation here, you can write just
```javascript
{
  [productSummary.nameContainer]: !showBorders,
}

```

Reply:
@Klynger done!